src/IonParserTextRaw.ts

```diff
@@ -1004,7 +1023,7 @@ export class ParserTextRaw {
   }
 
   get_value_as_string(t: number) : string {
-    var ii, ch, s = "";
+    let indice : number, ch : number, escaped, acceptComments : boolean, s : string = "";
```

Comment:
same comment on `indice`

also, more on a personal preference, if the variables declared are of *different* types then use separate declarations.